tools/pubsys/src/aws/validate_ssm/mod.rs

```diff
@@ -0,0 +1,732 @@
+//! The validate_ssm module owns the 'validate-ssm' subcommand and controls the process of validating
+//! SSM parameters and AMIs
+
+pub mod results;
+
+use self::results::{SsmValidationResult, SsmValidationResultStatus, SsmValidationResults};
+
+use super::ssm::ssm::get_parameters_by_prefix;
```

Comment:
Group these and let rust fmt sort them.

```suggestion
use self::results::{SsmValidationResult, SsmValidationResultStatus, SsmValidationResults};
use super::ssm::ssm::get_parameters_by_prefix;
```